imports/plugins/included/product-detail-simple/client/containers/variantListContainer.js

```diff
@@ -187,7 +204,7 @@ function composer(props, onData) {
   }
 
   let editable;
-
+  const products = props.product;
```

Comment:
Yes it exist in the if-else scope and also is been used by the onData function. For the variable to be accessible, it has to be outside the 2 scopes